### PR TITLE
JANITORIAL : Correct some more spelling mistakes

### PR DESCRIPTION
**Pull request by raziel-**

### CODE REVIEW

File: configure [hunk 1]
```diff
@@ -21750,7 +21750,7 @@ then :
           echo "*** you may also be able to get things to work by modifying LD_LIBRARY_PATH"
 else $as_nop
    echo "*** The test program failed to compile or link. See the file config.log for the"
-          echo "*** exact error that occured. This usually means ESD was incorrectly installed"
+          echo "*** exact error that occurred. This usually means ESD was incorrectly installed"
```
Comment (sezero) [inline configure:21753]: Drop this part: it actually comes from acinclude/esd.m4, and is not worth fixing.

File: test/configure [hunk 2]
```diff
@@ -4157,7 +4157,7 @@ then :
           echo "*** you may also be able to get things to work by modifying LD_LIBRARY_PATH"
 else $as_nop
    echo "*** The test program failed to compile or link. See the file config.log for the"
-          echo "*** exact error that occured. This usually means SDL was incorrectly installed"
+          echo "*** exact error that occurred. This usually means SDL was incorrectly installed"
```
Comment (sezero) [inline test/configure:4160]: Like above.

File: visualtest/configure [hunk 3]
```diff
@@ -3187,7 +3187,7 @@ if ac_fn_c_try_link "$LINENO"; then :
           echo "*** you may also be able to get things to work by modifying LD_LIBRARY_PATH"
 else
    echo "*** The test program failed to compile or link. See the file config.log for the"
-          echo "*** exact error that occured. This usually means SDL was incorrectly installed"
+          echo "*** exact error that occurred. This usually means SDL was incorrectly installed"
```
Comment (sezero) [inline visualtest/configure:3190]: Like above.

**Comment by raziel-** [comment]
Hmm, I've never dropped any changes.

Is it sufficient to change it back to what it was?

**Comment by icculus** [comment]
Yeah, just make a new commit to change it back and push; we're probably going to squash this all into one commit at the end.

**Comment by raziel-** [comment]
Done...I think?

**Comment by sezero** [comment]
> Done...I think?

Looks like it. No objections from me.

**Pull request merged**